packages/node-resolve/test/root-dir.js

```diff
@@ -7,11 +7,9 @@ const { testBundle } = require('../../../util/test');
 
 const { nodeResolve } = require('..');
 
-process.chdir(join(__dirname, 'fixtures', 'monorepo-dedupe', 'packages', 'package-a'));
-
 test('deduplicates modules from the given root directory', async (t) => {
   const bundle = await rollup({
-    input: 'index.js',
+    input: './packages/package-a/index.js',
```

Comment:
this was causing an error because the input could not be found. It should be relative to the `rootDir` right?